events/events.go

```diff
@@ -196,3 +204,22 @@ func (e *Events) unsubscribe(l listener) bool {
 	e.mu.Unlock()
 	return false
 }
+
+func GetContainerId(eng *engine.Engine, name string) string {
+	var buf bytes.Buffer
+	job := eng.Job("container_inspect", name)
+
+	var outStream, errStream io.Writer
+
+	outStream = &buf
+	errStream = outStream
+	job.Stdout.Set(outStream)
+	job.Stderr.Set(errStream)
```

Comment:
I doubt we need the errStream in this case. If there's an error, it should be the return value of `job.Run()`